app/requests_handlers.py

```diff
@@ -591,8 +596,12 @@ async def post(self, request):
                     raise ex
 
                 if request_type == 'paper-form':
-                    raise HTTPFound(
-                        request.app.router['RequestFormSentPost:get'].url_for(display_region=display_region))
+                    if 'request-name-address-large-print' in data:
```

Comment:
Further down, on about line 614, there is another if statement with special handling for "request_type == 'paper-form'". 
Just wondering if this also need a large print check, or if it can always use 'RequestFormCancelled:get'

Reply:
I think it's OK, as they are cancelling at the point where they get asked about whether they want large print, so the text on the cancel page is OK for both - it only splits on the confirmation page so that it is explicit in the confirmation as to what they are getting